pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/queries/ResourceBasedQueriesTest.java

```diff
@@ -73,6 +73,7 @@ public class ResourceBasedQueriesTest extends QueryRunnerTestBase {
   private static final String FILE_FILTER_PROPERTY = "pinot.fileFilter";
   private static final String IGNORE_FILTER_PROPERTY = "pinot.runIgnored";
   private static final int DEFAULT_NUM_PARTITIONS = 4;
+  private static final boolean REPORT_IGNORES = true;
```

Comment:
can we separate this one from this PR?

Reply:
Sure

Reply:
I've rollbacked most of these changes in 3dd08862ed9e5c2355f029276675ffc2179be886